Doc/extending/extending.rst

```diff
@@ -68,6 +69,9 @@ the module and a copyright notice if you like).
    headers on some systems, you *must* include :file:`Python.h` before any standard
    headers are included.
 
+   It is recommended to always define ``PY_SSIZE_T_CLEAN`` before including
+   ``Python.h``.  See :ref:`parsetuple` what this macro is.
```

Comment:
Apologies, I had suggested a change in the second line earlier, but I didn't know how to get it to render correctly in this window to show both lines.

```suggestion
   ``Python.h``.  See :ref:`parsetuple` for a description of this macro.
```